### ISSUE TITLE
Only the first 30 ops will run on the GPU, and the remaining 1 on the CPU

### ISSUE THREAD

**Issue by LukeChow** [tensorflow/tensorflow#31720]
I am building my codes with tensorflow_lite_gpu.framework on iPhone 7. The codes is just like Google recommends:

```
_model = FlatBufferModel::BuildFromFile(modelPathCString);
        ops::builtin::BuiltinOpResolver resolver;
        InterpreterBuilder(*_model, resolver)(&_interpreter);
        _delegate = NewGpuDelegate(nullptr);  // default config
        _interpreter->ModifyGraphWithDelegate(_delegate);
...//Other codes
```
When I use the model mobilenet_v1_1.0_224.tflite which Google provides, I get warning: `WARNING: 25 cannot be handled by this delegate.  Only the first 30 ops will run on the GPU, and the remaining 1 on the CPU` from console after excuting `_interpreter->ModifyGraphWithDelegate(_delegate)`. But the model deeplabv3_257_mv_gpu.tflite is good which Google provides too. As for my own model, the result is like mobilenet_v1_1.0_224.tflite does. Someone please help me.

PS: Here is my podfile: 
```
platform :ios, '10.0'
target "SpeechExample" do
pod 'TensorFlowLiteGpuExperimental'
end
```

**Comment by impjdi** [comment]
@LukeChow 

25 in the error message is probably referring to the op code 25, which is SOFTMAX.  It's been a while, but IIRC there's been some problems with SOFTMAX which should be solved by now.  However, the bad part is that there has been no recent cocoapod release with the fix in it.  Is it possible for you to try it out with the latest master branch of TF?

**Comment by LukeChow** [comment]
> @LukeChow
> 
> 25 in the error message is probably referring to the op code 25, which is SOFTMAX. It's been a while, but IIRC there's been some problems with SOFTMAX which should be solved by now. However, the bad part is that there has been no recent cocoapod release with the fix in it. Is it possible for you to try it out with the latest master branch of TF?

I am trying to build Tensorflow master source code. By the way, with my own
 model, I get warning `WARNING: 82 cannot be handled by this delegate`, which is Reduce Max op defined in file builtin_ops.h. But my model does not contain this op. I uploaded an attachment that contains information of layers used in my model. Could you please help me. Thank you very much. [layers.txt](https://github.com/tensorflow/tensorflow/files/3524494/layers.txt)

**Comment by yxchng** [comment]
@impjdi Where can I get the operator op code list? I need it to debug my code.

**Comment by impjdi** [comment]
@yxchng you can take a look at `delegates/gpu/common/model_builder.cc`  There is a huge switch-case near the end of the file.

**Comment by impjdi** [comment]
@LukeChow Sorry, this thread dropped off my radar.  It's possible that TOCO replaces certain graph patterns with a different op.

**Comment by sanatmpa1** [comment]
@LukeChow,

We are checking to see if you still need help on this issue. We recommend that you upgrade to `2.6` which is latest stable version of TF and let us know if the issue still persists in newer versions. we will get you the right help.Thanks!
